lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -92,3 +103,15 @@
         }
     }
 }
+
+.adf-info-drawer-layout-content .adf-content-metadata-card .adf-metadata-properties .mat-expansion-panel {
+    border: 1px solid var(--theme-metadata-property-panel-border-color);
+    border-radius: 12px;
+    margin-bottom: 12px;
+}
+
+.adf-content-metadata-card {
+    .mat-card:not([class*='mat-elevation-z']) {
```

Comment:
I think you should not be touching the internal `mat-elevation` here, can it be just `.mat-card` ?

Reply:
I have updated this